### ISSUE TITLE
Report datatables can't be browsed by keyboard. Missing WCAG 2.1 - 2.1.1 criterion.

### ISSUE THREAD

**Issue by tassoman** [piwik/piwik#20996]
## Context

When you're visiting [a report page represented by a data-table with actions](https://demo.matomo.cloud/index.php?module=CoreHome&action=index&idSite=1&period=day&date=yesterday#?idSite=1&period=day&date=yesterday&category=General_Actions&subcategory=General_Pages), keyboard navigation is uneffective.

Success criterion 2.1.1 of WCAG 2.1 requires the page must be browsable without the mouse.

![immagine](https://github.com/matomo-org/matomo/assets/92727/d2d9a050-eef1-4e5b-8821-74993efce938)

## Expected Behavior

We need two a11y anchors "skip to content" and "skip to menu".
Skip to content, should bring you at report title, then navigation should continue by listing the report's rows.
Report's rows should be all selectable. And their actions too. (buttons inside the row)
Expandable rows should expand by keyboard.    
Expanding a row should let you continue accessing "child" rows.  

## Current Behavior

When you choose link "skip to content", you got skipped ad page's title (#20869 #20913) then left pane menu steals the focus from the report's page. (in demo website, before this, focus gets stolen by upper side "demo bar").

When you get inside the report, you don't have any evidence where the focus is.

When you get inside the report, you're only able to select rows having links `a` tags who brings you outside the application with no warning.  
Leafs without link, are not selectable, so It's impossible to reach their action buttons, too. (attached image)

When you get inside the report, branch nodes identified with a ➕ plus or ➖ minus icon, can't be selected. And can't be expanded. Their leafs too. Their action buttons neither.

Every hovering action button of every row is not accessible by keyboard.

## Steps to Reproduce (for Bugs)

1. open [demo report](https://demo.matomo.cloud/index.php?module=CoreHome&action=index&idSite=1&period=day&date=yesterday#main)
2. try keyboard by using TAB key to tab forward, then SHIFT+TAB to go back. 

## Your Environment
* Matomo Version: 4.x
* PHP Version: 8.1
* Browser: any
* Operating System: any
